src/app/shared/filter.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from "@angular/core";
+import { ITech } from '../pages/tech/tech.service';
+
+export type classCheckbox = Array<{ name: string; isSelected: boolean }>;
+@Injectable({
+    providedIn: 'root'
+})
+
+export class FilterService {
+    filteredTech: ITech[] = [];
+    class: classCheckbox = [];
+    tech: ITech[] = [];
+
+    filterByName(e: any): ITech[] {
```

Comment:
can we make it so this doesn't reorder the tech list by job (class). i can add a id "column" to the data if we need.